ci/lib/templates.lib.txt

```diff
@@ -90,7 +90,7 @@ run-clang-tidy -j $(nproc) -quiet
 
 (@ def clang_format_bash_task(): -@)
 set -ueo pipefail
-export GEODE_HOME=$(pwd)/geode/expanded/apache-geode-$(cat geode/version)
+export GEODE_HOME=$(pwd)/geode
```

Comment:
Why set GEODE_HOME here, and use -DGEODE_ROOT in cmake configure line below?  It'd be great to do this consistently, probably using cmake variable in both cases.

Reply:
Sort of out of scope for this PR. I agree making them consistent would be nice.